packages/form/tests/integration/components/cf-field/input-test.js

```diff
@@ -122,6 +122,68 @@ module("Integration | Component | cf-field/input", function (hooks) {
     assert.dom("input[type=radio][value='option-1']").isChecked();
   });
 
+  test("it renders options of a radio field vertically", async function (assert) {
+    assert.expect(1);
+
+    this.set("noop", () => {});
+
+    await render(hbs`
+      {{cf-field/input
+        onSave=noop
+        field=(hash
+          options=(array
+            (hash slug="option-1" label="Option 1")
+            (hash slug="option-2" label="Option 2")
+          )
+          question=(hash
+            raw=(hash
+              __typename="ChoiceQuestion"
+              meta=(hash
+                vertical=true
+              )
+            )
+          )
+          answer=(hash
+            value=(array "option-1")
+          )
+        )
+      }}
+    `);
+
+    assert.dom(".uk-flex").exists();
+  });
+
+  test("it renders options of a radio field horizontally", async function (assert) {
+    assert.expect(1);
+
+    this.set("noop", () => {});
+
+    await render(hbs`
+      {{cf-field/input
+        onSave=noop
+        field=(hash
+          options=(array
+            (hash slug="option-1" label="Option 1")
+            (hash slug="option-2" label="Option 2")
+          )
+          question=(hash
+            raw=(hash
+              __typename="ChoiceQuestion"
+              meta=(hash
+                vertical=false
+              )
+            )
+          )
+          answer=(hash
+            value=(array "option-1")
+          )
+        )
+      }}
+    `);
+
+    assert.dom(".uk-flex").doesNotExist();
+  });
```

Comment:
This doesn't belong in the general field input test but in the respective input type test. Also, use angle brackets. Don't just copy paste other tests.

Reply:
I moved the tests into the respective input type tests.